rfc-030-auditing.md

```diff
@@ -0,0 +1,41 @@
+# Introduce auditing on all Rails projects
+
+## Summary
+
+We should routinely use auditing in our projects. For those projects which will
+undergo a Service Assessment, this is a requirement; for others it is good 
+practice to ensure confidence and security in the data we collect & store.
+
+## Problem
+
+For all projects undergoing a [GOV.UK Service Assessment](https://www.gov.uk/service-manual/service-assessments), 
+there is a requirement for non-repudiation - 
+["nothing should happen in a system that can’t be traced back to a responsible person"](https://www.gov.uk/service-manual/technology/securing-your-information#how-to-assess-information-security)
+
+All projects which are destined to undergo a Service Assessment should have
+auditing capabilities built-in from the start. For projects which are not
+likely to be assessed, it is still prudent to be able to trace data changes.
+
+It is easier to factor in auditing from the start of a project. If auditing is
+not considered until after a project has gone into private beta, we run the 
+risk of an incident occurring which would have been easier to trace & resolve
+if auditing was already in place.
+
+## Proposal
+
+The Rails Template project SHOULD include a mechanism for auditing by default.
```

Comment:
Should we clarify that the default CRUD actions should be audited by default? We're doing custom event tracking but that's hard to set any defaults for.